### DISCUSSION TITLE
Index not queryable in elasticsearch, can't get at bro/zeek records

### DISCUSSION THREAD

**Discussion by sei-dupdyke** [Security-Onion-Solutions/securityonion#6783] [Unsupported Versions]
Hi, we're trying to query bro/zeek records out of ElasticSearch (ES), and we typically do this on indexes matching `*:so-*`, but on this instance of Security Onion (SO) those indexes aren’t there, even though the default Index Pattern of the same format shows in Kibana.

I believe these are playbook indices. If so, can they be turned on somehow from python?

We see data with _index values that match the `*:so-*` pattern, but can’t seem to query it from python. Is there a way to enable the index to be available for querying? Or is it possible to query ES data from python without an index (sorry if this is more of an ES than an SO issue).

The code looks a bit like this (snipped for brevity... With these two versions, the `*:so-*` indices don't show:

```python
for index_name in es.indices.get_alias("*:so-*"):
  print(index_name)

for index_name in es.indices.get_alias("*"):
  print(index_name)
```

So try to query directly:

```python
items = es.search(size=10000)
for item in items:
  if "so" in item["_index"]:
    print("a record we want to use")
```

If I can make that index available to query or query data regardless of index, either is fine (but the latter probably is not performant?) - any help greatly appreciated, thanks.

**Comment by weslambert** [comment]
Are you querying the right node or in the right manner? What kind of deployment are you running? If you are attempting to query the manager and not querying across clusters, then you won't see the data you expect.
